apify-docs-theme/src/theme/DocBreadcrumbs/index.js

```diff
@@ -1,9 +1,9 @@
 import Link from '@docusaurus/Link';
-import { ThemeClassNames } from '@docusaurus/theme-common';
 import {
     useSidebarBreadcrumbs,
-    useHomePageRoute,
-} from '@docusaurus/theme-common/internal';
+} from '@docusaurus/plugin-content-docs/client';
+import { ThemeClassNames } from '@docusaurus/theme-common';
+import { useHomePageRoute } from '@docusaurus/theme-common/internal';
```

Comment:
i thought the `/internal` imports are no longer allowed?

Reply:
From what I understood, the Docusaurus team removed some `theme-common` package dependencies, as they didn't make sense from the "big-picture" POV. The `theme-common/internal` module was reexporting some of the symbols from those dependencies, which then indeed stopped working.

The `useHomePageRoute` hook is defined inside of `theme-common`, though, so it still works. Honestly, I don't really like importing something called `internal` either, but... as long as it works, I guess I'm fine?

Reply:
yeah sure, that's why I gave you the checkmark right ahead, build is passing, so it's fine.